config/database.yml.example

```diff
@@ -28,3 +30,4 @@ production:
   encoding: utf8
   pool: 5
   timeout: 5000
+  strict: false
```

Comment:
Same as above

Reply:
This was added by @jywarren in the update process